velox/exec/Window.cpp

```diff
@@ -368,6 +368,8 @@ void Window::callApplyForPartitionRows(
               isStartBound ? rawPeerStarts : rawPeerEnds;
           std::copy(rawPeerBuffer, rawPeerBuffer + numRows, frameBoundBuffer);
         } else {
+          // Fills the frameBound buffer with increasing values of row indices
```

Comment:
Nit -- Reframe "Fills the frameBound buffer with increasing value of row indices (corresponding to CURRENT ROW)  from the startRow of the current output buffer. The startRow has to be adjusted relative to the partition start row."